manifest.webapp

```diff
@@ -100,7 +100,12 @@
     "daccMonthlyCO2": {
       "type": "node",
       "file": "services/daccMonthlyCO2/coachco2.js",
-      "trigger": "@every 720h"
+      "trigger": "@monthly on the 1"
```

Comment:
wanted this change? If yes, should it be in another commit?

Reply:
Yes it is wanted, and yes it should be. It is now :) 
ff768380443ff5d9a40f73def3e8b0fa7f2fec72